src/Microsoft.Azure.Batch.SoftwareEntitlement/JwtPropertyProvider.cs

```diff
@@ -44,26 +44,26 @@ public Errorable<DateTimeOffset> IssuedAt()
         }
 
         /// <summary>
-        /// Gets the earliest moment at which the entitlement is active from the 'nbf' claim on the JWT.
+        /// Gets the earliest moment at which the token is active from the 'nbf' claim on the JWT.
         /// </summary>
         public Errorable<DateTimeOffset> NotBefore()
             => Errorable.Success(new DateTimeOffset(_jwt.ValidFrom));
 
         /// <summary>
-        /// Gets the latest moment at which the entitlement is active from the 'exp' claim on the JWT.
+        /// Gets the latest moment at which the token is active from the 'exp' claim on the JWT.
         /// </summary>
         public Errorable<DateTimeOffset> NotAfter()
             => Errorable.Success(new DateTimeOffset(_jwt.ValidTo));
 
         /// <summary>
-        /// Gets the audience for whom the entitlement is intended from the 'aud' claim on the JWT, or
+        /// Gets the audience for whom the token is intended from the 'aud' claim on the JWT, or
         /// an error if there are zero or many such claims.
         /// </summary>
         public Errorable<string> Audience()
         {
             // We don't expect multiple audiences to appear in the token
             var audiences = _jwt.Audiences.ToList();
-            if (audiences == null || !audiences.Any())
+            if (!audiences.Any())
```

Comment:
You're right that `audiences` can never be null - but I suspect `_jwt.Audiences` might be, requiring this instead:

``` csharp
var audiences = _jwt.Audiences?.ToList();
if (audiences is null || !audiences.Any())
```

Reply:
It (`JwtSecurityToken.Audiences`) can't be null, but just in case the implementation changes I'll put the check back in.